libraries/AP_NavEKF3/AP_NavEKF3_MagFusion.cpp

```diff
@@ -1054,6 +1093,10 @@ bool NavEKF3_core::fuseEulerYaw(yawFusionMethod method)
         innovYaw = 0.0f;
         break;
 
+    case yawFusionMethod::EXTNAV:
+        // both external sensor yaw and last yaw when static are stored in yawAngDataDelayed.yawAng
+        innovYaw = wrap_PI(yawAngPredicted - extNavYawAngDataDelayed.yawAng);
```

Comment:
Last yaw when static is being written to  yawAngDataStatic

Reply:
txs for the review, removed the 3 comments.